services/QuillLMS/app/queries/snapshots/period_query.rb

```diff
@@ -4,7 +4,7 @@ module Snapshots
   class PeriodQuery < ::QuillBigQuery::Query
     attr_accessor :timeframe_start, :timeframe_end, :school_ids, :grades
 
-    def initialize(timeframe_start, timeframe_end, school_ids, grades = nil, options = {})
+    def initialize(timeframe_start, timeframe_end, school_ids, grades = nil, options: {})
```

Comment:
I don't have strong feelings either way, but curious why this change? Changes to protocol for using the runner (the only use case) from `runner: runner` to `options: {runner: runner}`. NBD, just wondering if I'm missing something.